tests/behaviour/steps/inspectcli.py

```diff
@@ -556,5 +565,22 @@ def step_impl(context):
         context.result_dataset_value_counts
     )
     assert result_dataset_value_counts is not None
-
-    assert result_dataset_value_counts.by_measure_and_unit_val_counts_df.empty == True
+    
+    expected_df = pd.DataFrame(
+        [
+            {
+                "Measure": "Some Measure",
+                "Unit": "qb-id-10003.csv#unit/some-unit",
+                "Count": 3,
+            },
+                {
+                "Measure": "Some Other Measure",
+                "Unit": "qb-id-10003.csv#unit/percent",
```

Comment:
I think we're missing a step from the units/measures pivoting functionality. Once we've generated the unit's URI, we should be running a SPARQL query to find out whether we have a label for the unit in the RDF (and using that instead of the URI).

Can you put this into a tidy up ticket to be done after this PR is merged in?